### ISSUE TITLE
how to run_params from an outside of the repo

### ISSUE THREAD

**Issue by alxndrkalinin** [cytomining/evalzoo#14]
I wanted to define a relative [data_path](https://github.com/broadinstitute/position-effect-correction/blob/cda9e242aebea8031bd01a5a358a2c3507f9f3cb/2.evalzoo/params/params_orf_group.yaml#L3) in a config to make it user-independent (i.e. avoid overriding results folder and configs on Github), but seems like some paths in `evalzoo` are defined locally. 

E.g., defining `data_path: "../1.load/output"` and trying to run from another repo gives following:
```R
setwd("matric")
source("run_param.R")
setwd("../../position-effect-correction/2.evalzoo")
run_param("params/params_orf_group.yaml")
```
gives
```
Error in abs_path(input) : 
  The file '0.knit-notebooks.Rmd' does not exist.
In addition: Warning message:
In normalizePath(x, winslash = winslash, mustWork = must_work) :
  path[1]="0.knit-notebooks.Rmd": No such file or directory
```

I was able to make it work with `data_path: "../../position-effect-correction/1.load/output"` and 
```R
run_param("../../position-effect-correction/2.evalzoo/params/params_orf_group.yaml", results_root_dir = "../../position-effect-correction/2.evalzoo/results/")
```
however, it assumes both repos being located in the same directory, so doesn't fully address issues above.

**Comment by shntnu** [comment]
Oops – I certainly put in no effort into this being more than barely usable :D

> both repos being located in the same directory,

Can you clarify what you mean by "both repos"?

**Comment by alxndrkalinin** [comment]
Oh sorry, I just meant if your `evalzoo` and `position-effect-correction` were not checked out to the same directory, then `data_path: "../../position-effect-correction/1.load/output"` will be incorrect, because it's a relative path from a working directory (that _has_ to be inside evalzoo) to a file inside `position-effect-correction`.

Basically, when you create a new config for calculating metrics on a dataset, `data_path` inside this config always has to account for the relative location of of this dataset to `evalzoo`.

Does this make sense?

**Comment by shntnu** [comment]
Ah got it

So in this [param](https://github.com/broadinstitute/position-effect-correction/blob/cda9e242aebea8031bd01a5a358a2c3507f9f3cb/2.evalzoo/params/params_orf_group.yaml) file, you wish you could have specified 

https://github.com/broadinstitute/position-effect-correction/blob/cda9e242aebea8031bd01a5a358a2c3507f9f3cb/2.evalzoo/params/params_orf_group.yaml#LL3C11-L3C11

as 

```
data_path: "../1.load/output"
```
(Now I understand your first comment)

**Comment by alxndrkalinin** [comment]
Yep, exactly. And I think it comes down to being able to run `run_params` from outside of `evalzoo/matric`.

**Comment by shntnu** [comment]
Got it, and that's where the current strategy of using notebooks as functions gets a bit messy :D I'll keep this open for now and see if there's an easy fix
